src/current/cockroachcloud/production-checklist.md

```diff
@@ -6,9 +6,63 @@ cloud: true
 
-## Follow the SQL Best Practices
+Under the Shared Responsibility Model, Cockroach Labs is responsible for the following tasks: 
+
+- Cluster and cloud service availability and reliability
+- Maintenance and security of hardware and operating systems
+- Database and security patches 
+- Cluster backups 
+
+The customer is responsible for the following tasks:
+
+- Estimating workload and sizing the cluster 
+- Scaling clusters based on workload
+- Ensuring sufficient disk, compute, and memory capacity for each cluster
+- Monitoring cluster health and application performance
+- Ensuring that the workload is distributed appropriately across the nodes of the cluster
+- Performance tuning of SQL queries and schema
+- Initiating major version upgrades and selecting maintenance windows for patch releases 
+- (Optional) Taking customer-owned backups
+
+This page provides important recommendations for CockroachDB {{ site.data.products.cloud }} production tasks for which  the customer is responsible.
+
+## Deployment options
+
+When planning your deployment, it is important to carefully review and choose the [deployment options](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/choose-a-deployment-option) that best meet your scale, cost, security, and resiliency requirements. 
+
+Make sure your cluster has sufficient storage, CPU, and memory to handle the workload. The general formula to calculate the storage requirement is as follows:
+ 
+`raw data (storage, in GB) * replication factor (3 by default) * remove 40% to account for compression (0.6) * headroom (1.5-2)`
+
+For an example, see [Plan your Dedicated cluster]({% link cockroachcloud/plan-your-cluster.md %}).
+
+## Topology patterns
+
+When planning your deployment, it is important to carefully review and choose the [topology patterns](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/topology-patterns) that best meet your latency and resiliency requirements. This is especially crucial for multi-region deployments.
+
+## Cluster management
+
+You can create and manage CockroachDB {{ site.data.products.cloud }} clusters using the [Cloud Console](http://cockroachlabs.cloud), [Cloud API]({% link cockroachcloud/cloud-api.md %}), [ccloud CLI]({% link cockroachcloud/ccloud-get-started.md %}), or the [Terraform provider]({% link cockroachcloud/provision-a-cluster-with-terraform.md %}).
+
+## Network authorization
+
+CockroachDB {{ site.data.products.cloud }} requires you to authorize the networks that can access the cluster in order to prevent denial-of-service and brute force password attacks. During the application development phase, you might have authorized only your local machine’s network. To move into production, you need to authorize your application servers' networks.
```

Comment:
```suggestion
CockroachDB {{ site.data.products.cloud }} requires you to authorize the networks that can access the cluster in order to prevent denial-of-service and brute force password attacks. During the application development phase, you might have authorized only your local machine’s network. To move into production, you need to authorize your the networks used by your application servers.
```